Src/SemanticComparison/ProxyGenerator.cs

```diff
@@ -173,13 +183,10 @@ private static void BuildMethodGetHashCode<TDestination>(TypeBuilder type)
             method.SetReturnType(typeof(int));
 
             int derivedGetHashCode = 135;
-            MethodInfo getHashCode = typeof(TDestination).GetMethod(

```

Comment:
Notice a change here - now only public method will be matched. That should be fine as `GetHashCode` is public.

Reply:
@moodmosaic Will you be able to find some time in the nearby future and follow up on this one so we can proceed further? ☺️

Reply:
I'd rather define an `Operating System` with the properties of the old one just to make the pull request smaller. This pull request, as-is, in order to review it carefully, requires me to find a contiguous time, which unfortunately I won't have until the end of this year.

Reply:
@moodmosaic Done! Defined a copy of the `OperatingSystem` class, so now no changes in unit tests are present.

You are fine with the changes now to proceed further?